packages/react/src/components/FileUploader/FileUploader.tsx

```diff
@@ -342,6 +344,6 @@ FileUploader.propTypes = {
    * sizes.
    */
   size: PropTypes.oneOf(['sm', 'md', 'lg']),
-};
+} as PropTypes.ValidationMap<FileUploaderProps>;
```

Comment:
I was getting TS errors and added this as a workaround:

accept: `Type '(string | null | undefined)[]' is not assignable to type 'string[]'`

filenameStatus: `Type 'string' is not assignable to type '"edit" | "complete" | "uploading"'.`